x/emissions/keeper/keeper.go

```diff
@@ -526,6 +533,52 @@ func (k *Keeper) DeleteUnfulfilledReputerNonces(ctx context.Context, topicId Top
 	return k.unfulfilledReputerNonces.Set(ctx, topicId, types.ReputerRequestNonces{})
 }
 
+/// INCLUSIONS
+
+func (k *Keeper) GetCountInfererInclusionsInTopic(ctx context.Context, topicId TopicId, inferer ActorId) (uint64, error) {
+	key := collections.Join(topicId, inferer)
+	count, err := k.countInfererInclusionsInTopic.Get(ctx, key)
+	if err != nil {
+		if errors.Is(err, collections.ErrNotFound) {
+			return 0, nil
+		}
+		return 0, err
+	}
+	return count, nil
+}
+
+func (k *Keeper) IncrementCountInfererInclusionsInTopic(ctx context.Context, topicId TopicId, inferer ActorId) error {
```

Comment:
We can rely on these never being called from multiple goroutines, right?

Reply:
ok rely is a strong word, but we definitely try not to use any goroutines anywhere in the code

it was something we got dinged for in a previous security audit